mlir/lib/Dialect/Bufferization/Transforms/OneShotModuleBufferize.cpp

```diff
@@ -255,6 +255,9 @@ static void equivalenceAnalysis(FunctionOpInterface funcOp,
                                 OneShotAnalysisState &state,
                                 FuncAnalysisState &funcState) {
   funcOp->walk([&](CallOpInterface callOp) {
+    if (isa<func::CallIndirectOp>(callOp))
+      return WalkResult::skip();
+
     FunctionOpInterface calledFunction = getCalledFunction(callOp);
     assert(calledFunction && "could not retrieved called FunctionOpInterface");
```

Comment:
```suggestion
    FunctionOpInterface calledFunction = getCalledFunction(callOp);
    if (!calledFunction)
      return WalkResult::skip();
```

Reply:
This looks good. Skipping call ops for which we cannot find the target should be fine.